src/Lykke.JobTriggers/Lykke.JobTriggers.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Lykke.Common" Version="5.1.0" />
+    <PackageReference Include="Lykke.Common" Version="7.0.0-beta.15" />
```

Comment:
beta?

Reply:
This will be replaced with release version before PR merging